tsod/mvdetectors.py

```diff
@@ -0,0 +1,95 @@
+import pandas as pd
+import numpy as np
+from typing import Union
+
+from .base import Detector
+
+
+class MVRangeDetector(Detector):
+    """
+    Detect values outside range.
+
+    If one or more time series is out of range, is is detected as an anomaly. Note that this implies that the same range
+    is used for all time series.
+
+    Parameters
+    ----------
+    min_value : float
```

Comment:
min and max values must be able to be varied for each individual time-series, e.g. for  simultaneous measurement of temperature and humidity, you need different ranges. But keep the option to provide a single value applied for all timeseries.

Reply:
Thanks for having a look quickly! Sure, I can do that. The intent was to have a very simple method with simple inputs. But the idea of allowing for both options (either same min/max for all or specified indvidually) is also reasonable. 

The question is of course how many options to pack into one thing versus constructing another detector with this specific purpose. I think I might actually prefer pursuing the original intent and instead adding another detector that has min/max for each time series. It would also avoid some if/else in e.g. fitting, which will keep the code clearer.

Let me know what you think though, if you still prefer combining both options in one detector we can also do that.

Reply:
I think we should follow common conventions in NumPy and Pandas [e.g `pandas.DataFrame.clip`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.clip.html)

The clip method accepts both scalar and array as input.
![image](https://user-images.githubusercontent.com/614215/152315920-0c822cee-bffe-45a3-975f-df70a9c631fe.png)

Reply:
good point!

Reply:
I was able to use broadcasting to avoid if/else and have added changes that allow different ranges for different time series.